SETUP.md

```diff
@@ -146,13 +143,15 @@ After setup, read about our [code styleguide](./STYLEGUIDE.md), our [test suites
     1. `echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list`
     1. `sudo apt-get update && sudo apt-get install yarn=1.16.0-1`
     1. `yarn --version` Double check the version of yarn is correct.
+1. Make it so that you can run apps tests locally
+    1. `export CHROME_BIN=$(which chromium-browser)`
```

Comment:
Should we say to add this to your bash profile?

Reply:
Also, does this just work by default on Mac because Chrome is already installed? And the default `CHROME_BIN` points to it already?

Reply:
> Should we say to add this to your bash profile?

Yes, good catch! updated.

> Also, does this just work by default on Mac because Chrome is already installed? And the default `CHROME_BIN` points to it already?

Apparently Mac OS does not come with Chrome installed. I'll add a note about this to the OS X setup instructions.

This seems to just work once Chrome is installed, but not via `CHROME_BIN`. Since this was working (once Chrome is installed) without additional configuration, I didn't investigate further.